packages/core/src/main/conversation/ConversationService.test.node.ts

```diff
@@ -1,6 +1,6 @@
 /*
  * Wire
- * Copyright (C) 2018 Wire Swiss GmbH
+ * Copyright (C) 2022 Wire Swiss GmbH
```

Comment:
this should not be updated actually, only add current copyright for new files, on old updated files they should reflect the original copyright date
```suggestion
 * Copyright (C) 2018 Wire Swiss GmbH
```